tests/test_gw_repeater.sh

```diff
@@ -77,12 +77,10 @@ main() {
 
     [ "$START_REPEATER" = "true" ] && {
         index=0
-        no_vendor="-n"
         for repeater in $REPEATER_NAMES; do
             status "Start Repeater (Remote Agent): $repeater"
-            ${rootdir}/tools/docker/run.sh -u ${UNIQUE_ID} ${VERBOSE_OPT} ${FORCE_OPT} start-agent -d -n ${repeater} -m aa:bb:cc:$index$index -- $no_vendor "$@"
+            ${rootdir}/tools/docker/run.sh -u ${UNIQUE_ID} ${VERBOSE_OPT} ${FORCE_OPT} start-agent -d -n ${repeater} -m aa:bb:cc:$index$index -- "$@"
             index=$((index+1))
-            no_vendor=
         done
```

Comment:
So, this means we no longer have a test of an agent that acts as a non-prplmesh agent. Is that acceptable?

Maybe it's better to keep all this and just force the controller_is_prplmesh option to false if the no_vendor_specific option is set.

Reply:
I think it is acceptable since we are testing agent certification on a nightly basis with non-prplmesh controllers and controller certification with non-prplmesh agents.